### ISSUE TITLE
Permissions on /etc/cif/rules directory causing problems within docker container

### ISSUE THREAD

**Issue by deanpemberton** [csirtgadgets/verbose-robot#14]
# Do you have an active https://csirtg.io/support account ? :)
No

# Expected behavior and actual behavior.
When running inside a docker container - csirtg-smrt should start up and parse feed rule files.
Instead it gives errors about unsupported file type for the /etc/cif/rules/default directory.

# Steps to reporduce the problem

$ ./docker_build.sh
$ sh ./docker/run.sh

# Relevant logs as a result of the actual behavior

Monitoring the logs for the container shows that csirtg-fm is in a loop giving an error about unsupported file type.

~~~
2018-06-21 00:20:30,723 DEBG 'csirtg-fm' stderr output:
2018-06-21 00:20:30,721 - INFO - csirtg_fm.cli[54] - starting run...

2018-06-21 00:20:30,920 DEBG 'csirtg-fm' stderr output:
2018-06-21 00:20:30,919 - INFO - csirtg_fm.rule[35] - processing /etc/cif/rules/default

2018-06-21 00:20:30,930 DEBG 'csirtg-fm' stderr output:
2018-06-21 00:20:30,920 - ERROR - csirtg_fm.rule[39] - unsupported file type: /etc/cif/rules/default
2018-06-21 00:20:30,920 - INFO - csirtg_fm.cli[120] - cleaning up
2018-06-21 00:20:30,920 - INFO - csirtg_fm.cli[124] - finished run
2018-06-21 00:20:30,920 - INFO - csirtg_fm.cli[126] - sleeping...

2018-06-21 00:20:35,716 DEBG 'csirtg-fm' stderr output:
2018-06-21 00:20:35,713 - DEBUG - csirtg_fm.cli[217] - forking process...

2018-06-21 00:20:35,722 DEBG 'csirtg-fm' stderr output:
2018-06-21 00:20:35,721 - INFO - csirtg_fm.cli[54] - starting run...

2018-06-21 00:20:35,884 DEBG 'csirtg-fm' stderr output:
2018-06-21 00:20:35,884 - INFO - csirtg_fm.rule[35] - processing /etc/cif/rules/default

2018-06-21 00:20:35,885 DEBG 'csirtg-fm' stderr output:
2018-06-21 00:20:35,885 - ERROR - csirtg_fm.rule[39] - unsupported file type: /etc/cif/rules/default
~~~

Looking at the directory it appears to have no execute permission

~~~
root@f84c22ebbebc:/etc/cif/rules# ls -l
total 4
drw-rw---- 2 cif cif 4096 Jun 21 01:32 default
root@f84c22ebbebc:/etc/cif/rules# cd ..
root@f84c22ebbebc:/etc/cif# ls -l
total 4
drw-rw---- 1 cif cif 4096 Jun 21 01:31 rules
~~~

Changing this using the following command.

~~~
chmod -R 770 /etc/cif/rules
~~~

Causes csirtg-fm to wake up immediately and start processing feeds.

~~~
2018-06-21 01:43:45,548 DEBG 'csirtg-fm' stderr output:
2018-06-21 01:43:45,548 - INFO - csirtg_fm.cli[126] - sleeping...

2018-06-21 01:43:50,433 DEBG 'csirtg-fm' stderr output:
2018-06-21 01:43:50,429 - DEBUG - csirtg_fm.cli[217] - forking process...

2018-06-21 01:43:50,438 DEBG 'csirtg-fm' stderr output:
2018-06-21 01:43:50,437 - INFO - csirtg_fm.cli[54] - starting run...

2018-06-21 01:43:50,525 DEBG 'csirtg-fm' stderr output:
2018-06-21 01:43:50,525 - DEBUG - csirtg_fm.clients.http[82] - /var/lib/cif/dataplane.org

2018-06-21 01:43:50,534 DEBG 'csirtg-fm' stderr output:
2018-06-21 01:43:50,525 - DEBUG - csirtg_fm.clients.http[104] - /var/lib/cif/dataplane.org/sshpwauth.txt
2018-06-21 01:43:50,525 - DEBUG - csirtg_fm.clients.http[121] - CACHE /var/lib/cif/dataplane.org/sshpwauth.txt
2018-06-21 01:43:50,525 - DEBUG - csirtg_fm.clients.http[184] - cache size is 0, downloading...
2018-06-21 01:43:50,528 - DEBUG - urllib3.connectionpool[823] - Starting new HTTPS connection (1): dataplane.org:443

2018-06-21 01:43:51,400 DEBG 'csirtg-fm' stderr output:
2018-06-21 01:43:51,399 - DEBUG - urllib3.connectionpool[393] - https://dataplane.org:443 "GET /sshpwauth.txt HTTP/1.1" 200 None

2018-06-21 01:43:51,807 DEBG 'csirtg-fm' stderr output:
2018-06-21 01:43:51,807 - DEBUG - csirtg_fm.cli[89] - testing parser: /var/lib/cif/dataplane.org/sshpwauth.txt

2018-06-21 01:43:51,819 DEBG 'csirtg-fm' stderr output:
2018-06-21 01:43:51,818 - DEBUG - csirtg_fm.cli[91] - detected parser: pipe

2018-06-21 01:43:51,827 DEBG 'csirtg-fm' stderr output:
2018-06-21 01:43:51,826 - DEBUG - csirtg_fm.parsers.delim[32] - 3            |  MIT-GATEWAYS - Massachusetts I  |      18.82.1.150  |  2018-06-20 23:36:40  |  sshpwauth
~~~

# Specifications like the version of the project, operating system, or hardware.

$ docker --version
Docker version 18.05.0-ce, build f150324

**Comment by deanpemberton** [comment]
Changing 660 to 770 in the entrypoint file fixes this for me.

**Comment by wesyoung** [comment]
hrm- i wanna say i ran into that too in my testing but now i can't see if/where i fixed it (maybe i was just testing on a local docker instance and didn't push the changes). or maybe i added 0660 instead of 0770 by accident (more likely)?

mind sending a PR, will get it merged.. ty!

**Comment by wesyoung** [comment]
fixed in 4.0.0a3
